recipes/rmlui/4.x/conanfile.py

```diff
@@ -59,8 +63,8 @@ def configure(self):
 
```

Comment:
Just before this line you could do `self.options.rm_safe("fPIC")` instead of `del self.options.fPIC`.